tests/test_sample.py

```diff
@@ -0,0 +1,10 @@
+import pytest
+from mujtaba_charm.utils.sample import hello
+
+def test_hello():
+    assert hello("john") == "hello john!"
+    assert hello() == "hello world!"
+    assert hello("john") == "hello world!"
```

Comment:
what is this 3rd test case supposed to check?

Reply:
It is an invalid case, hello("john") should not == "hello world!", I checked whether this test actually fails like its supposed to.